config.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
We will merge this to main (so to use mcstaging on main), then perform switch of aemshop backend data, then delete this config from main (thus "falling back" to mcprod) in a new PR.